typings/react-native-code-push.d.ts

```diff
@@ -65,6 +73,16 @@ export interface CodePushOptions extends SyncOptions {
      * It can be the version from package.json (or another source).
      */
     runtimeVersion: string;
+    /**
+     * Specifies which versioning mode to use.    
+     * You should provide `customVersioning` option if you want to use `"custom"` versioning mode.
+     */
+    versioningMode: VersioningMode;
+    /**
+     * Specifies custom versioning policy.    
+     * Custom versioning is applied only when `versioningMode` is `"custom"`.
+     */
+    customVersioning: Versioning;
```

Comment:
@changwoolab 
I think `customVersioning` should be optional.

And maybe `versioningMode`, too.
How about encouraging users to conform to SemVer versioning?
In this case, we can specify the default mode as `"semver"`.